Software/Python/easygopigo3.py

```diff
@@ -445,7 +445,7 @@ def target_reached(self, left_target_degrees, right_target_degrees):
         else:
             return False
 
-    def reset_encoders(self):
+    def reset_encoders(self, blocking=True):
         """
         | Resets both the encoders back to **0**.
```

Comment:
Can this method raise exceptions? If so, we need to list them.

Also, the `blocking` parameter has to be described. 
Maybe some more details as to what are the effects of this method should be brought in.

Reply:
No exception.
I realized as I was going to bed that I had forgotten to document the blocking parameter. :)
Now done. Good catch!